ci/do_circle_ci.sh

```diff
@@ -13,8 +13,8 @@ export ENVOY_SRCDIR="$(pwd)"
 # hard code this (basically due to how docker works).
 export NUM_CPUS=8
 
-# Circle does not currently support IPv6 so we don't test it here. We will add and remove Travis
-# when it is supported.
+# CircleCI doesn't support IPv6 by default, so we run all tests with IPv4, and
+# a limited subset with IPv6 using machine: true and do_circle_ci_ipv6_tests.sh
```

Comment:
maybe write this as "machine: true" and link to https://circleci.com/docs/2.0/executor-types/#using-machine